src/patterns/dosdp-patterns/infectious_disease_and_sequelae_by_agent.yaml

```diff
@@ -0,0 +1,50 @@
+pattern_name: infectious_disease_and_sequelae_by_agent
+
+pattern_iri: http://purl.obolibrary.org/obo/mondo/patterns/infectious_disease_and_sequelae_by_agent.yaml
+
+description: 'Infectious diseases can be classified by the infections agent, such
+  as bacteria, coronavirus, etc, that causes the disease, and their associated sequelae or aftereffects.
+
+  Examples: [COVID-19 and sequelae](http://purl.obolibrary.org/obo/MONDO_0100318)'
+
+contributors:
+- https://orcid.org/0000-0002-6601-2165
+- https://orcid.org/0000-0001-5208-3432
+
+classes:
+  disease: MONDO:0000001
+  organism: NCBITaxon:1
+  infectious disease: MONDO:0005550
+
+relations:
+  disease arises from feature: RO:0004022
+
+annotationProperties:
+  exact_synonym: oio:hasExactSynonym
+  related_synonym: oio:hasRelatedSynonym
+
+vars:
+  agent: '''organism'''
+  infectious disease: '''infectious disease'''
+
+name:
+  text: '%s infectious disease and sequelae'
+  vars:
+  - agent
+
+annotations:
+- annotationProperty: broad_synonym
+  text: % infectious disease
+  vars:
+  - agent
+
+def:
+  text: A disease and associated sequelae caused by infection with %s.
+  vars:
+  - agent
+
+equivalentTo:
+  text: '''%s'' and ''disease arises from feature'' some %s'
```

Comment:
the grouping class should be a union of

 - D
 - disease and 'disease arises from feature' some D

Reply:
would COVID-19 and sequelae look like this?

![image](https://user-images.githubusercontent.com/6722114/113360543-dbaf9d80-92fe-11eb-8c77-e96c197dad92.png)

Reply:
or is it meant to be like this?
![image](https://user-images.githubusercontent.com/6722114/113361707-65f90100-9301-11eb-9e48-f0cea09b4e9c.png)

but this causes the class to be equiv to COVID-19

Reply:
or is it meant to be an 'or'?
![image](https://user-images.githubusercontent.com/6722114/113361771-8923b080-9301-11eb-827d-7a42a872b331.png)